src/util/feefrac.h

```diff
@@ -53,10 +57,19 @@ struct FeeFrac
 #ifdef __SIZEOF_INT128__
     static inline __int128 Mul(int64_t a, int32_t b) noexcept
     {
-        // If __int128 is available, use 128-bit wide multiply.
+        // If __int128 is available, use 128-bit wide multiplication.
         return __int128{a} * b;
     }
+#elif defined(_MSC_VER) && defined(_M_X64)
+    static inline std::pair<int64_t, uint64_t> Mul(int64_t a, int32_t b) noexcept
+    {
+        // On 64-bit MSVC, use _mul128 intrinsic for wide multiplication.
+        std::pair<int64_t, uint64_t> ret;
+        ret.second = _mul128(a, b, &ret.first);
```

Comment:
To help other reviewers:
```c++
__int64 _mul128(
   __int64 Multiplier,
   __int64 Multiplicand,
   __int64 *HighProduct
);

Return value
The low 64 bits of the product.
```

So this should result in:
ret = <high, low>

Which matches the other impls.